qutebrowser/misc/sessions.py

```diff
@@ -459,6 +464,34 @@ def list_sessions(self):
                 sessions.append(base)
         return sorted(sessions)
 
+    def add_url_to_session(self, name):
+        """Appends the current tab's URL to a named session."""
+        data = self._load_session_file(name)
+
+        if data is None:
+            raise SessionError("Got empty session file")
```

Comment:
What about building up a new session file using this command, is that something you want to support?